src/main/java/api/ApiRequestManager.java

```diff
@@ -52,6 +52,11 @@
         return executeWithBody(endpoint, pathParams, entity, ApiMethod.POST);
     }
 
+    public static <T> ApiResponse create(final String endpoint, final Map<String, String> pathParams, final String entity)
+            throws JsonProcessingException {
+        return executeWithBody(endpoint, pathParams, entity, ApiMethod.POST);
+    }
```

Comment:
create and update are similar methods, you can use the same method since it only changes method type "POST / PATCH"